packages/clay-card/src/CardWithInfo.tsx

```diff
@@ -135,20 +135,31 @@ export const ClayCardWithInfo: React.FunctionComponent<IProps> = ({
 		image: displayType === 'image' || imgProps,
 	};
 
+	const contentSymbol = useMemo(() => {
+		if (!symbol) {
+			if (isCardType.image) {
+				return 'camera';
+			}
+
+			return 'documents-and-media';
+		}
+
+		return symbol;
+	}, [isCardType]);
```

Comment:
I don't think you want `useMemo` for this. The cost of memoization is probably close to the cost of the operation you're trying to memoize, and possibly greater than it:

- Without memoization = one or two simple `if` statements, zero or one property accesses.
- With memoization = an array instantiation, a function call, iteration over the array with an `Object.is()` call for each item in it (1).

Similar analysis for `stickSymbol` below.

Reply:
Better now?

Reply:
> Better now?

![never-enough](https://user-images.githubusercontent.com/7074/100467293-80f90180-30d2-11eb-8c84-ab1fad8c59fb.gif)

Instead of making functions and calling them, just do:

```js
const contentSymbol = symbol
    ? symbol
    : isCardType.image
    ? 'camera'
    : 'documents-and-media';

const stickeSymbol = isCardType.image ? 'document-image' : 'document-default';
```

You avoid the function call overhead (creating a stack frame, calling, returning etc). These simple ternary expressions are so fast that you can consider them to be "free".

Reply:
Done